### PR TITLE
Added Toast, Rating and Grid

### PR DESCRIPTION
**Pull request by ashishkatiyar2404**
Screenshot (779)
Screenshot (780)
Screenshot (781)

### CODE REVIEW

File: Components/rating.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.rating__star i {
+  color: var(--gray-color);
+  font-size: 2.4rem;
+  cursor: pointer;
+}
+.rating__star .checked,
+.rating__star i:hover {
+  color: var(--gold-color);
+}
```
Comment (ShrutiLakhani) [inline Components/rating.css:9]: Everything looks good. LGTM!!!

File: Components/toast.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  font-weight: 400;
+}
+.toast__items p,
+.toast__items i {
+  cursor: pointer;
+}
+.toast__items span,
+.toast__items p {
+  margin-right: 2rem;
+}
```
Comment (ShrutiLakhani) [inline Components/toast.css:24]: LGTM!

File: Components/simplified-grid.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+}
+.three__item-box1 {
+  background-color: var(--primary-color);
+}
+.three__item-box2 {
+  background-color: var(--gray-color);
+}
+.three__item-box3 {
+  background-color: var(--footer-icon);
+}
```
Comment (ShrutiLakhani) [inline Components/simplified-grid.css:33]: Good to merge!

**Review by ShrutiLakhani — APPROVED** [review]
Everything looks good. Easy to understand code. LGTM!!!

File: Components/badge.css [hunk 4]
```diff
@@ -88,13 +88,18 @@
...
+  position: relative;
+}
+
+.badges__icon-link span {
   position: absolute;
+  top: -2rem;
+  right: -0.5rem;
 }
 
 .badges__icon-img a > i {
```
Comment (pankaj-prog) [inline Components/badge.css:105]: you can be consistent here, u have used diff way to target the same element

**Review by pankaj-prog — APPROVED** [review]

File: Components/toast.css [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  width: max-content;
+  border-radius: 1rem;
+}
+.toast__items span,
+.toast__items i {
+  color: var(--white-color);
+}
+.toast__items p {
+  color: var(--nav-btn-bg-color);
+  font-weight: 400;
```
[the remark this answers is not in the document]
Comment (ashishkatiyar2404) [reply Components/toast.css:15]: yes... i have to refactor the root... will change

**Pull request merged**